Library/Homebrew/dependency.rb

```diff
@@ -168,6 +168,16 @@ def merge_temporality(deps)
         [] # Means both build and runtime dependency.
       end
     end
+
+    def expand_dependency(dependency)
```

Comment:
You don't need to explicitly pass the block to the method unless you are referencing it in the method. That is, unless you're calling `block.call` or the like you're fine without it. It's implicit otherwise and you can interact with it via `block_given?`, `yield`, etc.